ooyala_player/ooyala_player.py

```diff
@@ -196,6 +197,8 @@ def student_view(self, context):
             'height': self.height,
             'autoplay': self.autoplay,
             'config_url': json_config_url,
+            'publish_completion_url': self.runtime.handler_url(self, 'publish_completion', '').rstrip('/?'),
```

Comment:
@macornwell we don't need to pass this `publish_completion_url` here instead we call `publish_completion` handler from JS like this
```
url: runtime.handlerUrl(element, 'publish_completion'),
```

Reply:
Good catch. Makes it much simpler.